### PR TITLE
Improvement in Docs

### PR DESCRIPTION
**Pull request by Kartik494**
This PR Fix wording for  Install and Set Up kubectl 
Fixes: #27410

### CODE REVIEW

**Comment by Kartik494** [comment]
@sftim PTAL

File: content/en/docs/tasks/tools/install-kubectl-macos.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ card:
 
 ## {{% heading "prerequisites" %}}
 
-You must use a kubectl version that is within one minor version difference of your cluster.
-For example, a v1.2 client should work with v1.1, v1.2, and v1.3 master.
+You must use a kubectl version that is within one minor version difference of your cluster. For example, a v{{< skew latestVersion >}} client can communicate with v{{< skew prevMinorVersion >}}, and v{{< skew nextMinorVersion >}} control planes.
```
Comment (Aut0R3V) [inline content/en/docs/tasks/tools/install-kubectl-macos.md:15]: @Kartik494 I think when @sftim meant mentioning the version numbers.

Comment (sftim) [reply content/en/docs/tasks/tools/install-kubectl-macos.md:15]: > version numbers

I did, but that's an optional extra improvement only. The change from “master” to “control plane” is the essential detail.

**Review by sftim — CHANGES_REQUESTED** [review]
Make sure to document that a `v{{< skew latestVersion >}}` client talking to a `v{{< skew latestVersion >}}` control plane is supported too!

**Comment by Kartik494** [comment]
Hi @sftim  if i get it right you are asking here to to modify the language as ?

`You must use a kubectl version that is within one minor version difference of your cluster. For example, a v{{< skew latestVersion >}} client can communicate with v{{< skew latestVersion >}},  v{{< skew prevMinorVersion >}} and v{{< skew nextMinorVersion >}} control planes.`

**Comment by Kartik494** [comment]
Hi @sftim i have added a commit related to this. Please let me know is this the required changes.

Thanks

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27425--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -12,8 +12,7 @@ card:
 
 ## {{% heading "prerequisites" %}}
 
-You must use a kubectl version that is within one minor version difference of your cluster.
-For example, a v1.2 client should work with v1.1, v1.2, and v1.3 master.
+You must use a kubectl version that is within one minor version difference of your cluster. For example, a v{{< skew latestVersion >}} client can communicate with v{{< skew latestVersion >}}, v{{< skew prevMinorVersion >}} and v{{< skew nextMinorVersion >}} control planes.
```
Comment (kbhawkey) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:15]: Hi @Kartik494 . Thanks for contributing.
Looking at the preview pages, what do you think about listing the versions in order,
v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, and v{{< skew nextMinorVersion >}}?

Comment (Kartik494) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:15]: @kbhawkey Thanks for the input , i will update it accordingly

Comment (Kartik494) [reply content/en/docs/tasks/tools/install-kubectl-linux.md]: @kbhawkey   i have addressed the changes . PTAL

**Comment by kbhawkey** [comment]
Thanks for the updates @Kartik494 .
@sftim, Are there remaining questions?
/lgtm

**Pull request merged**